apps/showcase/jest.config.js

```diff
@@ -1,4 +1,4 @@
-const { getJestModuleNameMapper } = require('@o3r/dev-tools');
+const { getJestModuleNameMapper } = require('@o3r/workspace');
```

Comment:
Are we sure we want to move it to `workspace`?
If we do so, it meets that the all the generated app/libs should have a dep to `workspace`.
Which is not what we want actually.

I understand that we need to move it out of the `dev-tools` package and that the `core` is not an option (becasue build in ESM).
What is about `testing`?